grpc.go

```diff
@@ -81,9 +81,53 @@ func handleError(wait bool, err error) error {
 	return err
 }
 
-// Online resources, nbResources specifies the maximum number of resources to online.
-// If nbResources is <= 0 then there is no limit and all resources are connected.
+// Offline resources, nbResources specifies the maximum number of resources to online.
```

Comment:
Offline resources, ... maximum number of resources to **online** ?

Reply:
yes, thanks for the pointing